test/DotNetEnv.Tests/EnvConfigurationTests.cs

```diff
@@ -67,6 +92,7 @@ public void AddSourceToBuilderAndLoadMultiWithNoClobber()
                 .Build();
 
             Assert.Equal("Toni", this.configuration["NAME"]);
+            Assert.Equal("ENV value", configuration["ENVVAR_TEST"]);
```

Comment:
Without the change you will get `overridden`, because EnvVars are just ignored here.
But if you have an interpolation using `ENVVAR_TEST` it eveluates to `ENV value`, because Interpolation uses the EnvVars.
![grafik](https://github.com/tonerdo/dotnet-env/assets/52062680/64f477c6-ba5f-4712-ba9f-670f7b2b2269)

That leads to the next point I will mention, but definitely not solving in this PR here: if you disable `SetEnvVars`, the Interpolation stops working. (has to be proven with tests of course, but having a look into the code it checks for Environment only, so it can't work if SetEnvVars is disabled imho)

Reply:
Because I mentioned it, I investigated how to show that interpolation is not working without `SetEnvVars`.
You just need to add `LoadOptions.NoEnvVars()` in `EnvConfigurationTests-->AddSourceToBuilderAndParseInterpolatedTest`:
```
this.configuration = new ConfigurationBuilder()
                .AddDotNetEnv("./.env_embedded", LoadOptions.NoEnvVars())
                .Build();
```
The test fails now with the first interpolation as shown here:
![grafik](https://github.com/tonerdo/dotnet-env/assets/52062680/216d7595-ad5a-4f96-8eb1-cfc6e56d1567)

Not a concern in this PR, but the reason why I want to add this PR-fix here first --> not too much changes in a single PR